### PR TITLE
fix: replace get() with getProperty()

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
All usages of get() method were replaced by calling getProperty() method.

### CODE REVIEW

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
You can also delete `deprecated` get(...) method from `store.service.ts`.

File: libs/perun/login/src/lib/login-screen-base/login-screen-base.component.ts [hunk 1]
```diff
@@ -25,11 +25,11 @@ export class LoginScreenBaseComponent implements OnInit {
...
   ngOnInit(): void {
-    this.headerBackgroundColor = this.storeService.get('theme', 'nav_bg_color') as string;
-    this.headerTextColor = this.storeService.get('theme', 'nav_text_color') as string;
-    this.logo = this.sanitizer.bypassSecurityTrustHtml(this.storeService.get('logo') as string);
+    this.headerBackgroundColor = this.storeService.getProperty('theme').nav_bg_color;
+    this.headerTextColor = this.storeService.getProperty('theme').nav_text_color;
+    this.logo = this.sanitizer.bypassSecurityTrustHtml(this.storeService.getProperty('logo'));
     this.textColor = this.headerTitle
-      ? (this.storeService.get('theme', 'header_text_color') as string)
+      ? this.storeService.getProperty('theme').header_text_color
```
Comment (HejdaJakub) [inline libs/perun/login/src/lib/login-screen-base/login-screen-base.component.ts:32]: You can delete `header_text_color` from `ConfigProperties.ts`.
```suggestion
      ? this.storeService.getProperty('theme').nav_text_color
```

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.ts [hunk 2]
```diff
@@ -8,51 +8,29 @@ import { ReportIssueDialogComponent } from '../report-issue-dialog/report-issue-
...
-    'footer',
-    'github_backend_releases'
-  ) as string;
-  iconColor: string = this.storeService.get('theme', 'footer_icon_color') as string;
-  bgColor: string = this.storeService.get('theme', 'footer_bg_color') as string;
+  headersTextColor: string = this.storeService.getProperty('theme').footer_headers_text_color;
+  linksTextColor: string = this.storeService.getProperty('theme').footer_links_text_color;
+  githubRepository: string = this.storeService.getProperty('footer').github_releases;
+  githubBackendRepository: string = this.storeService.getProperty('footer').github_backend_releases;
+  iconColor: string = this.storeService.getProperty('theme').footer_icon_color;
```
Comment (HejdaJakub) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.ts:31]: I couldn't find any history of the `footer_icon_color` property. Variable `iconColor` is NOT used in HTML, so I think you can delete this line and the property `footer_icon_color` can be deleted from `ConfigProperties.ts`.
```suggestion
```

File: libs/perun/services/src/lib/store.service.ts [hunk 3]
```diff
@@ -62,73 +62,41 @@ export class StoreService {
...
-          }
-          currentValue = currentValue[keys[i]];
-        }
+  addMissingValuesToProperty<T extends keyof PerunConfig>(
+    value: PerunConfig[T],
+    defaultValue: PerunConfig[T]
+  ): PerunConfig[T] {
+    if (typeof value === 'object' && !Array.isArray(value) && value) {
+      for (const key of Object.keys(defaultValue)) {
+        defaultValue[key] = this.addMissingValuesToProperty(value[key], defaultValue[key]);
```
Comment (HejdaJakub) [inline libs/perun/services/src/lib/store.service.ts:95]: I think you can use something like this to fix eslint errors.
```suggestion
        defaultValue[key] = this.addMissingValuesToProperty(
           value[key] as PerunConfig[T], 
           defaultValue[key] as PerunConfig[T]
        );
```

File: libs/perun/services/src/lib/store.service.ts [hunk 4]
```diff
@@ -62,73 +62,41 @@ export class StoreService {
...
-            // console.error('Missing value in default config: ' + keys);
-            break;
-          }
-          currentValue = currentValue[keys[i]];
-        }
+  addMissingValuesToProperty<T extends keyof PerunConfig>(
+    value: PerunConfig[T],
+    defaultValue: PerunConfig[T]
+  ): PerunConfig[T] {
+    if (typeof value === 'object' && !Array.isArray(value) && value) {
```
Comment (HejdaJakub) [inline libs/perun/services/src/lib/store.service.ts:93]: There is a little "problem" with an array of objects properties like the `custom_labels` property. On the other hand properties of this type are just about labels (`ProfileAttribute[]`, `ProfileExtService[]` and `ProfileCustomLabel[]`) so I don't think that it is a big problem at all. IMHO it's more about a bad configuration, if someone put just a part of this config type to the `instanceConfig`, what do you think (@xkostka2)?

**Review by HejdaJakub — CHANGES_REQUESTED** [review]

File: libs/perun/services/src/lib/store.service.ts (hunk 3 above)
Comment (bodnara) [reply libs/perun/services/src/lib/store.service.ts:95]: I already said this to @sarkapalkovicova but the issue here is that on consecutive passes the values are no longer item of PerunConfig but some nested item. So the definition of function has to be changed.

File: libs/perun/services/src/lib/store.service.ts (hunk 4 above)
Comment (xkostka2) [reply libs/perun/services/src/lib/store.service.ts:93]: I agree, default values in array-of-objects properties are usually just examples of how to fill those properties.

**Review by bodnara — APPROVED** [review]

**Pull request merged**